src/components/message/message-card/MessageCardBase.tsx

```diff
@@ -68,25 +69,40 @@ export function MessageCardBase({
 }: MessageCardBaseProps) {
 	const { messageId, timestamp, messageType, sessionId, direction, bodyBase64, body } = message;
 
+	const iconsClassName = createBemBlock(
+		'message-card-icons',
+		isAttached ? 'attached' : null,
+		isBookmarked ? 'pinned' : null,
+	);
+
 	const renderInlineMessageInfo = () => {
 		if (viewType === MessageViewType.ASCII || viewType === MessageViewType.JSON) {
 			const formattedTimestamp = formatTime(timestampToNumber(timestamp));
+			const splittedMessageId = messageId.split(':');
+			const messageIdWithHighlightedSession = splittedMessageId.map(
+				splittedMessageId.length > 3
+					? extendedGetMessageIdWithHighlightedSession
+					: getMessageIdWithHighlightedSession,
+			);
 			return (
 				<>
+					{isExport && (
+						<Checkbox
```

Comment:
This checkbox shifts whole message list, i think we need to avoid it

Reply:
it is added according to https://www.figma.com/file/vB9EyLv1GBE3vJqy2Hxk08/TH2-New-Report-Style-Updates?node-id=2738%3A80177